.github/workflows/knative-style.yaml

```diff
@@ -209,54 +209,6 @@ jobs:
 
           echo '::endgroup::'
 
-      - name: Redundant Format
```

Comment:
Oh good. Are we also going to go back and fix all the places we updated to the new worse format to appease the lint check? 😩